src/main/resources/templates/settingsForSingleProject.vm

```diff
@@ -167,8 +167,8 @@ $webResourceManager.requireResource("de.uhd.ifi.se.decision.management.jira:styl
 				classifyAllIssuesInProjectButton.addEventListener("click", function() {
 					if (confirm('Are you sure you want to classify all issue comments? This might take some time if the project is big.')) {
 						AJS.progressBars.setIndeterminate("#progress-bar");
-                		classifyWholeProject('$projectKey');
-                		AJS.progressBars.update("#progress-bar", 1.0);
+                		var isSucceeded = classifyWholeProject('$projectKey'); 
```

Comment:
[Alternative] I also tried to update the progress bar via a callback function but this did not work. [/Alternative]
[Con] AJS.progressBars.update("#progress-bar", ...) could not be found by the nested callback function. [/Con]